packages/t/TypewriterJS.json

```diff
@@ -15,13 +15,13 @@
   "license": "MIT",
   "author": "Tameem Safi <@tameemsafi> (https://safi.me.uk/)",
   "autoupdate": {
-    "source": "git",
-    "target": "https://github.com/tameemsafi/typewriterjs.git",
+    "source": "npm",
+    "target": "typewriter-effect",
     "fileMap": [
       {
-        "basePath": "",
+        "basePath": "dist",
         "files": [
-          "typewriter*"
+          "**/*"
```

Comment:
```suggestion
          "*.js"
```
Let's use a [more specific glob](https://www.digitalocean.com/community/tools/glob?comments=true&glob=dist%2F%2A.js&matches=true&tests=%2F%2F%20Imported%20from%20typewriter-effect%20NPM%20package&tests=LICENSE&tests=preview.gif&tests=examples%2Fauto-type-out-all-strings%2Findex.html&tests=examples%2Fcall-callback%2Findex.html&tests=examples%2Fchained-example%2Findex.html&tests=examples%2Fchange-cursor%2Findex.html&tests=examples%2Fchat-bot-example%2Findex.html&tests=examples%2Fdelete-all-speed-change%2Findex.html&tests=examples%2Floop%2Findex.html&tests=examples%2Fstrings-with-html-nested%2Findex.html&tests=examples%2Fstrings-with-html-tags%2Findex.html&tests=src%2Futils%2Fadd-styles.js&tests=src%2Futils%2F__test__%2Fadd-styles.spec.js&tests=babel.config.js&tests=webpack%2Fcommon.config.babel.js&tests=src%2Fcore%2Fconstants.js&tests=dist%2Fcore.js&tests=webpack%2Fdevelopment.config.babel.js&tests=src%2Futils%2Fdoes-string-contain-html-tag.js&tests=src%2Futils%2F__test__%2Fdoes-string-contain-html-tag.spec.js&tests=src%2Futils%2Fget-dom-element-from-string.js&tests=src%2Futils%2F__test__%2Fget-dom-element-from-string.spec.js&tests=src%2Futils%2Fget-random-integer.js&tests=src%2Futils%2F__test__%2Fget-random-integer.spec.js&tests=src%2Fcore%2Findex.js&tests=src%2Freact%2Findex.js&tests=src%2Futils%2Findex.js&tests=webpack%2Fproduction.config.babel.js&tests=dist%2Freact.js&tests=src%2Fcore%2FTypewriter.js&tests=src%2Freact%2FTypewriter.js&tests=src%2Fcore%2F__test__%2FTypewriter.spec.js&tests=src%2Freact%2F__test__%2FTypewriter.spec.js&tests=package.json&tests=readme.md&tests=src%2Fcore%2F__test__%2F__snapshots__%2FTypewriter.spec.js.snap&tests=index.d.ts&tests=.circleci%2Fconfig.yml) so we aren't left open to potential abuse